examples/niscope/fetch-continuous.py

```diff
@@ -4,6 +4,9 @@
 # The gRPC API is built from the C API.  NI-SCOPE documentation is installed with the driver at:
 # C:\Program Files (x86)\IVI Foundation\IVI\Drivers\niScope\Documentation\English\Digitizers.chm
 #
+# A version of this .chm is available online at:
+# Link: https://zone.ni.com/reference/en-XX/help/370592AB-01/
```

Comment:
I guess we stopped posting help online after 16.1?

Reply:
Yeah, this is the latest available version for now.